integration/tests/basic/test_rpc_calls.py

```diff
@@ -174,9 +175,55 @@ def test_eth_gas_price(self, param: tp.Union[str, None]):
             response["result"]
         ), f"Invalid current gas price `{response['result']}` in wei"
 
+    @pytest.mark.parametrize(
+        "blockhash, tag1, tag2", [
```

Comment:
why do you need blockhash as parameter if it is always True

Reply:
Removed